packages/gatsby-plugin-clerk/src/GatsbyClerkProvider.tsx

```diff
@@ -7,6 +7,13 @@ import {
 import { navigate } from 'gatsby';
 import React from 'react';
 
+import { name, version } from '../package.json';
+
+const SDK_METADATA = {
+  name,
+  version,
+};
```

Comment:
We are importing from `package.json` directly here as this package isn't built via `tsup`, so we can't define the global variable at build-time.